enhancements/observability/cluster-observability-operator-logging-stack.md

```diff
@@ -0,0 +1,405 @@
+7. The COO tracks the collection components until they reach phase `ready`.
+
+### API Extensions
+
+#### OperatorConfig
+
+The `OperatorConfig` resource is the entry point to tell COO on which operator installations it should manage as long as one `LoggingStack` is managed too. It is a cluster-scoped resource.
+
+```yaml
+apiVersion: observability.openshift.io/v1alpha1
+kind: OperatorConfig
+metadata:
+  name: coo-config
+spec:
+  subscriptions:
+  - type: logging
+    installPlanApproval: Automatic
+    channel: stable-5.8
+```
+
+#### LoggingStack
+
+The `LoggingStack` is meant to be the minimum viable declaration of the logging stack possible, e.g. user provides refs to external dependencies like S3-compatible storage credentials, Certificate Authority bundles. It is a namespace-scoped resource with the limitation for now that only one can exist in `openshift-logging`.
+
+```yaml
+apiVersion: logging.observability.openshift.io/v1alpha1
+kind: LoggingStack
+metadata:
+  name: sample-logging-stack
+  namespace: openshift-logging
+spec:
+  monitoringSelector:
+    openshift.io/cluster-monitoring: "true"
+  storage:
+    size: 1x.extra-small
+    storageClassName: gp3-csi
+    storage:
+      secret:
+        name: aws-s3-credentials
+        type: s3
+```
+
+The `UIPlugin` resources are discussed in a [separate proposal](observability-ui-proposal) here.
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+N/A.
+
+#### Standalone Clusters
+
+This proposal is mainly targeting standalone clusters either on-premises or OpenShift Dedicated. Users upgrading from existing OpenShift Logging installation need to uninstall them first and re-install through the COO by following the above workflow. Doing so will enable to use the correlation component out of the box.
+
+#### Single-node Deployments or MicroShift
+
+N/A.
+
+### Implementation Details/Notes/Constraints
+
+#### Handling multiple LoggingStack resourceselector on the same cluster
+
+Historically the OpenShift Logging product allows the creation **only** of a single logging stack per cluster. This design detail represents a choice to preserve resources on a cluster rather than a limitation. Having multiple end-to-end stacks is rather a very expensive solution and has been corroborated by different means:
+- Soft tenancy combined with full RBAC-based controls on logs. This design choice renders the need for multiple expensive log storage installations obsolete.
+- Using namespaced `ClusterLogForwarder` resources to offers users with configuration options on which workloads should be collected as well as how to filter logs.
+
+Therefore it is possible to allow creating multiple `LoggingStack` resources on the same cluster with the following workflow:
+1. Create main `LoggingStack` resource in `openshift-logging` namespace that reconciles the operator installation as well as the resources `ClusterLogging` and `Lokistack`.
```

Comment:
Based upon our latest 6.0 discussions, I think this workflow is no longer valid since ClusterLogging does not exist